core/src/subgraph/context/mod.rs

```diff
@@ -58,6 +58,10 @@ impl SubgraphKeepAlive {
             self.sg_metrics.running_count.inc();
         }
     }
+
+    pub fn contains(&self, deployment_id: &DeploymentId) -> bool {
```

Comment:
nitpick: I would rename this to is_alive or is_running which seems better (subjectively) suited for "SubgraphKeepAlive"

Reply:
There are already `remove` and `insert` methods, and I thought `contains` would follow the same format :)